### PR TITLE
[FR] Add `User_title`

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal for the French translation of the wiki page `User_title`.

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

*Feel free to merge if the delay is too long for a review by a native.*

### CODE REVIEW

File: wiki/User_title/fr.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+| Spinner Sage | 120–179 |
+| Star Shooter | 180–259 |
+| Combo Commander | 260–499 |
+| Rhythm Incarnate | 500+ |
+
+Ces titres n'étaient visibles que sur les barres latérales des messages du forum (pas sur les profils des utilisateurs) et n'étaient pas affichés si un autre titre personnalisé était déjà défini auparavant.
+
+## Le saviez-vous ?
+
+- Les titres des utilisateurs sont appelés "titres de rang" dans le code du jeu. Ce nom est un vestige de l'époque où osu! était construit sur [phpBB] (https://www.phpbb.com/).
```
Comment (DoryanR) [inline wiki/User_title/fr.md:52]:
```suggestion
- Les titres des utilisateurs sont appelés "titres de rang" dans le code du jeu. Ce nom est un vestige de l'époque où osu! était construit sur [phpBB](https://www.phpbb.com/).
```

File: wiki/User_title/fr.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+tags:
+  - forum title
+  - forum titles
+  - rank
+  - titles
+  - user color
+  - user colour
+  - titre du forum
+  - titres du forum
+  - classement
```
Comment (Ganondorf-1) [inline wiki/User_title/fr.md:11]:
```suggestion
  - grade
```

File: wiki/User_title/fr.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+![Capture d'écran des informations du profil de Tom94](img/dev.png "Le titre \"osu!dev\" est visible en haut du profil de Tom94.")
+
+Un **titre d'utilisateur** est une courte étiquette sur la page de profil d'un utilisateur, définie par l'[équipe osu!](/wiki/People/The_Team). La plupart des titres indiquent l'appartenance à un [groupe d'utilisateurs](/wiki/People/The_Team#groupes-d'utilisateurs), mais certains utilisateurs ont obtenu des [titres plus uniques](/wiki/People/Users_with_unique_titles) pour des réalisations ou des contributions spécifiques.
+
+La couleur du titre d'un utilisateur est la même que celle de l'utilisateur, qui est généralement définie par l'appartenance à un groupe. Si l'utilisateur n'a pas de couleur définie, le titre est blanc.
+
+## Histoire
+
+### Titres du nombre de messages
```
Comment (Ganondorf-1) [inline wiki/User_title/fr.md:29]:
```suggestion
### Titres du nombre de posts
```

File: wiki/User_title/fr.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+![Capture d'écran de la barre latérale du poste de Damnae sur l'ancien site web](img/star-shooter.png "Le titre \"Star Shooter\" se trouve au-dessus du nombre de messages de Damnae.")
```
Comment (Ganondorf-1) [inline wiki/User_title/fr.md:31]:
```suggestion
![Capture d'écran de la barre latérale du poste de Damnae sur l'ancien site web](img/star-shooter.png "Le titre \"Star Shooter\" se trouve au-dessus du nombre de posts de Damnae.")
```

File: wiki/User_title/fr.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+Sur les forums de [l'ancien site d'osu!](https://old.ppy.sh), les utilisateurs pouvaient obtenir des titres en atteignant certains seuils de nombre de messages, comme suit :
```
Comment (Ganondorf-1) [inline wiki/User_title/fr.md:33]:
```suggestion
Sur les forums de [l'ancien site d'osu!](https://old.ppy.sh), les utilisateurs pouvaient obtenir des titres en atteignant certains seuils de nombre de posts, comme suit : 
```

File: wiki/User_title/fr.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+## Histoire
+
+### Titres du nombre de messages
+
+![Capture d'écran de la barre latérale du poste de Damnae sur l'ancien site web](img/star-shooter.png "Le titre \"Star Shooter\" se trouve au-dessus du nombre de messages de Damnae.")
+
+Sur les forums de [l'ancien site d'osu!](https://old.ppy.sh), les utilisateurs pouvaient obtenir des titres en atteignant certains seuils de nombre de messages, comme suit : 
+
+| Titre du forum | Nombre total de messages |
```
Comment (Ganondorf-1) [inline wiki/User_title/fr.md:35]:
```suggestion
| Titre du forum | Nombre total de posts |
```

File: wiki/User_title/fr.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+| Spinner Sage | 120–179 |
+| Star Shooter | 180–259 |
+| Combo Commander | 260–499 |
+| Rhythm Incarnate | 500+ |
+
+Ces titres n'étaient visibles que sur les barres latérales des messages du forum (pas sur les profils des utilisateurs) et n'étaient pas affichés si un autre titre personnalisé était déjà défini auparavant.
+
+## Le saviez-vous ?
+
+- Les titres des utilisateurs sont appelés "titres de rang" dans le code du jeu. Ce nom est un vestige de l'époque où osu! était construit sur [phpBB](https://www.phpbb.com/).
```
Comment (Ganondorf-1) [inline wiki/User_title/fr.md:52]:
```suggestion
- Les titres des utilisateurs sont appelés "rank titles" dans le code du jeu. Ce nom est un vestige de l'époque où osu! était construit sur [phpBB](https://www.phpbb.com/).
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Comment by DoryanR** [comment]
@Ganondorf-1 recheck please

**Review by TicClick — APPROVED** [review]
merging with the assumption that one review was enough

**Pull request merged**